README.md

```diff
@@ -43,6 +43,12 @@ All Markdown files here are run through [tianon's fork of `markdownfmt`](https:/
 
 <!-- AUTOGENERATED TOC -->
 
+# How do I update an image's docs
+
+Edit the `content.md` for an image; not the `README.md` as it's auto-generated from the contents of the other files in that repo. To see the changes to the `README.md`, run `./update.sh myimage` from the repo root, but do not add the `README.md` changes to your pull request. See also `markdownfmt.sh` point [below](#how-do-i-add-a-new-images-docs).
+
+After opening your Pull Request the changes will be checked by an automated `markdownfmt.sh` before it can be merged. A common issue is incorrect spacing such as with two lines missing an empty line between them (double-spaced)
```

Comment:
I think this sentence should probably end with a period:

```suggestion
After opening your Pull Request the changes will be checked by an automated `markdownfmt.sh` before it can be merged. A common issue is incorrect spacing such as with two lines missing an empty line between them (double-spaced).
```